Examples/Scripts/PostFreeSurferPipelineBatch-long.sh

```diff
@@ -0,0 +1,243 @@
+  job_list=()
+  for TP in ${Timepoints[@]}; do
+	echo "Running ppFS-long for timepoint: $TP"
+        job=$($queuing_command ${HCPPIPEDIR}/PostFreeSurfer/PostFreesurferPipelineLongPrep.sh --subject="$Subject" --path="$StudyFolder" \
+            --template="$Longitudinal_Template" --timepoints="$TP" --template_processing=0 --t1template="$T1wTemplate" \
+            --t1templatebrain="$T1wTemplateBrain" --t1template2mm="$T1wTemplate2mm" --t2template="T2wTemplate" \
+            --t2templatebrain="$T2wTemplateBrain" --t2template2mm="$T2wTemplate2mm" --templatemask="$TemplateMask" \
+            --template2mmmask="$Template2mmMask" --fnirtconfig="$FNIRTConfig" --freesurferlabels="$FreeSurferLabels")
+        echo "submitted timepoint job $job"
+        job_list+=("$job")
+  done
+  #fi
+  jl="${job_list[@]}"  
+  #Process template and finalize timepoints. This must wait until all timepoints are finished.
+  echo "Running ppFS-long for template $Template"
+  template_job=$($queuing_command -j ${jl// /,} ${HCPPIPEDIR}/PostFreeSurfer/PostFreesurferPipelineLongPrep.sh --subject="$Subject" --path="$StudyFolder" \
+	--template="$Longitudinal_Template" --timepoints="${Timepoint_list[i]}" --template_processing=1 --t1template="$T1wTemplate" \
+        --t1templatebrain="$T1wTemplateBrain" --t1template2mm="$T1wTemplate2mm" --t2template="T2wTemplate" \
+        --t2templatebrain="$T2wTemplateBrain" --t2template2mm="$T2wTemplate2mm" --templatemask="$TemplateMask" \
+        --template2mmmask="$Template2mmMask" --fnirtconfig="$FNIRTConfig" --freesurferlabels="$FreeSurferLabels")
+        echo "submitted template job $job"
+
+  echo "Template processing job $template_job will wait for timepoint jobs $jl"
+
+  ##########################################################################################
+  # PostFreesurferPipeline.sh processing
+  ##########################################################################################
+  echo "Timepoint processing, stage 1"
+  job_list=()
+  for Timepoint in ${Timepoints[@]}; do
+  	#input variables specific for timepoint mode (if any)  		  
+  	# ...
+	
+	#DEBUG
+	#continue
+	#process each timepoint
+	job=$($queuing_command -j $template_job "$HCPPIPEDIR"/PostFreeSurfer/PostFreeSurferPipeline.sh \
+	      --study-folder="$StudyFolder" \
+	      --subject="$Subject" \
+	      --surfatlasdir="$SurfaceAtlasDIR" \
+	      --grayordinatesdir="$GrayordinatesSpaceDIR" \
+	      --grayordinatesres="$GrayordinatesResolution" \
+	      --hiresmesh="$HighResMesh" \
+	      --lowresmesh="$LowResMeshes" \
+	      --subcortgraylabels="$SubcorticalGrayLabels" \
+	      --freesurferlabels="$FreeSurferLabels" \
+	      --refmyelinmaps="$ReferenceMyelinMaps" \
+	      --regname="$RegName" \
+	      --longitudinal_mode="TIMEPOINT_STAGE1" \
+	      --longitudinal_template="$LongitudinalTemplate" \
+	      --longitudinal_timepoint="$Timepoint")
+	job_list+=("$job")
+	#DEBUG
+	#break
+  done
+  jl="${job_list[@]}"
```

Comment:
```suggestion
  jl=$(IFS=','; echo "${job_list[*]}")
```
Then you don't need the `${jl// /,}` bit below.  Note, `*` makes a single string, `@` is for making separate arguments to a command or loop.